main.py

```diff
@@ -35,30 +35,28 @@
 BASE_SCHEMA = ['title', 'contributor', 'creator', 'subject', 'description', 'language', 'publisher']
 
 
-def get_oai_properties(base_url, shortname):
+def get_oai_properties(base_url, shortname,start_date,end_date):
     """ Makes a request to the provided base URL for the list of properties
 
         returns a dict with list of properties
     """
 
     try:
-        print
-        start_date = (date.today() - timedelta(1)).isoformat()
-        end_date = date.today().isoformat()
-        with vcr.use_cassette('../scrapi/tests/vcr/{}.yaml'.format(shortname)):
-            prop_url = base_url + '?verb=ListRecords&metadataPrefix=oai_dc&from={}&until={}'.format(start_date, end_date)
+        prop_url = base_url + '?verb=ListRecords&metadataPrefix=oai_dc&from={}&until={}'.format(start_date, end_date)
+
         print('requesting {}'.format(prop_url))
         prop_data_request = requests.get(prop_url)
         all_prop_content = etree.XML(prop_data_request.content)
         pre_names = all_prop_content.xpath('//ns0:metadata', namespaces=NAMESPACES)[0].getchildren()[0].getchildren()
-
         all_names = [name.tag.replace('{' + NAMESPACES['dc'] + '}', '') for name in pre_names]
         return list({name for name in all_names if name not in BASE_SCHEMA})
 
     # If anything at all goes wrong, just render a blank form...
     except Exception as e:
```

Comment:
If you're catching specific exceptions, do it here on the outer level instead of within the generic one. You can catch multiple exceptions at once